i18n/en.json

```diff
@@ -8170,4 +8338,4 @@
     "id": "web.incoming_webhook.user.app_error",
     "translation": "Couldn't find the user."
   }
-]
+]
```

Comment:
There is so much duplication being proposed in this translation file change, but I'm not sure how hard it would be to deal with it and if it's worth it.

Reply:
Yeah, there are some reusable strings, however, deciding where to reuse them in code makes things even more hard to follow; having them clearly separated in code and some duplication in the translation file also allows us to easily update the strings as needed.